cli/commands/common.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2016 IBM Corporation
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+package commands
+
+import (
+	"github.com/amalgam8/amalgam8/cli/terminal"
+	"github.com/amalgam8/amalgam8/cli/utils"
+	"github.com/urfave/cli"
+	"os"
+	"strings"
+)
+
+var (
+	// JSON .
+	JSON = strings.ToLower(utils.JSON)
+	// YAML .
+	YAML = strings.ToLower(utils.YAML)
+	// TABLE .
+	TABLE = "table"
+)
+
+// CommandTable .
```

Comment:
https://github.com/olekukonko/tablewriter may be an easier alternative than pretty printing code in commands